game_runner.rb

```diff
@@ -0,0 +1,10 @@
+class GameRunner
+  def self.run(*args)
+    GameRunner.new.run(*args)
+  end
+
+  def run(stdin)
+  end
+end
+
+GameRunner.run($stdin)
```

Comment:
When you run rspec, do you see the grid printed out alongside the test output?

Reply:
I did when I had it this way.